tests/cypress/specs/ui/enrollment.spec.js

```diff
@@ -1,13 +1,16 @@
 const agencies = require("../../fixtures/transit-agencies");
 
+const eligibility_confirm_url = "/eligibility/confirm";
```

Comment:
I noticed you removed usage of this variable in https://github.com/cal-itp/benefits/pull/493/commits/7c5e83eff093ec84ba366b44e06c698661e3d616, so we should remove it now, right?

```suggestion
```